src/core/transport/upnp.cc

```diff
@@ -275,57 +275,57 @@ void UPnP::Discover() {
 void UPnP::TryPortMapping(
     int type,
     int port) {
-  std::string type;
-  std::string port(std::to_string(port));
+  std::string strType;
```

Comment:
You only needed to change the case for `controlURL` because it is a library-defined member of miniupnpc's `UPNPUrls`. The rest of these were fine in your original change, so no need to revert.